app/views/notifier/invite.markerb

```diff
@@ -1 +1,8 @@
-<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/', :diaspora_id => @inviter.try(:diaspora_handle)) %>
+<%= t('.message',
+  invite_url: invite_code_url(@invitation_code),
+  diasporafoundation_url: 'https://diasporafoundation.org/',
+  user: @inviter.try(:diaspora_handle) == @inviter.try(:name) ?
```

Comment:
Sorry, but this is really unreadable. Also it feels wrong to first fallback to the diaspora ID and then ignore it again. It would be better to handle it directly. As I wrote in https://github.com/diaspora/diaspora/issues/6559#issuecomment-314927738 we should probably create a method, that returns `nil` when there is no name.